packages/openid4vc/src/openid4vc-issuer/OpenId4VcIssuerEvents.ts

```diff
@@ -0,0 +1,15 @@
+import type { OpenId4VcIssuanceSessionState } from './OpenId4VcIssuanceSessionState'
+import type { OpenId4VcIssuanceSessionRecord } from './repository'
+import type { BaseEvent } from '@credo-ts/core'
+
+export enum OpenId4VcIssuerEvents {
+  IssuanceSessionStateChanged = 'OpenId4VcIssuer.IssuanceSessionStateChanged',
```

Comment:
Not for now, but I think events that indicate the state change like we do with DIDComm protocols may be nice

Reply:
This is how we do it with DIDComm as well. What do you mean exactly?

Reply:
You're right, I was confused here. I do think having separate events is more fitting for OpenID and probably also for DIDComm now I think about it, but my comment was wrong.